content/blog/2020/07/2020-07-27-repository-signing-keys-changing.adoc

```diff
@@ -0,0 +1,82 @@
+---
+layout: post
+title: "Jenkins 2.235.3: New Linux Repository Signing Keys"
+tags:
+- announcement
+- linux
+- platform
+author: markewaite
+description: >
+    This article announces new Linux repository signing keys.
+    Repository signing keys will change with the release of Jenkins LTS 2.235.3.
+    The previous repository signing keys will not be used after the release of Jenkins LTS 2.235.3.
+    The repository signing keys will be changed for Debian/Ubuntu and CentOS/Red Hat repositories.
+opengraph:
+  image: /images/post-images/2020/07-repository-signing-keys-change/opengraph.png
```

Comment:
Are we allowed to use all the logos on this image?

Reply:
I reviewed the Red Hat and Ubuntu trademark usage guidelines and did not see any restriction that would forbid our use.  However, I am not a lawyer.  I'm willing to remove logos from the opengraph image if there are concerns.  The information in the blog post is much more valuable than the information in the opengraph image.